<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>129</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>432</width>
    <height>129</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>百度图片批量下载器</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="word_label">
         <property name="text">
          <string>关键词</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="word_lineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="num_label">
         <property name="text">
          <string>数量</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="num_spinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>60</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="size_label">
         <property name="text">
          <string>尺寸</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="total_radioButton">
         <property name="text">
          <string>全部</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="XL_radioButton">
         <property name="text">
          <string>特大</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="L_radioButton">
         <property name="text">
          <string>大</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="M_radioButton">
         <property name="text">
          <string>中</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="S_radioButton">
         <property name="text">
          <string>小</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="thread_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>线程</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="thread_spinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="dir_label">
         <property name="text">
          <string>存储目录</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dir_lineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dir_pushButton">
         <property name="text">
          <string>浏览</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="status_label">
         <property name="text">
          <string>状态空闲</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="download_pushButton">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>下载</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="src_pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>源码</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
